src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1349,4 +1351,119 @@ private static int indexOfSameAllele(final VariantContext vc, final Allele allel
 
         return -1;
     }
+
+    /**
+     * Add chromosome Counds (AC, AN and AF) to the VCF header lines
+     *
+     * @param headerLines the VCF header lines
+     */
+    public static void addChromosomeCounts(final Set<VCFHeaderLine> headerLines) {
```

Comment:
Is `addChromosomeCounts` the right name for this?  I think maybe `addAlleleCountsToHeader` is more accurate if a bit more wordy.